### ISSUE TITLE
More Effective Timer Sounds

### ISSUE THREAD

**Issue by Dontbtme** [sanderland/katrain#79]
Using  v1.2.0, timer' sounds currently inform us that:
_ A few seconds remain till the end of our byoyomi period.
_ 1 second remains till the end of our byoyomi period.
As a result, if you play after hearing the last sound, you may or may not have played on time: there's no way to know without directly looking at the timer, which would defeat the purpose of having sounds in my opinion.
Likewise, you can't know if you lost on time or not without taking a look at the timer, which is hard when you're focusing on the board. Chances are you'll continue playing until you realize that it's been a while since you heard the timer at all.

In short, to be effective, I believe sounds should inform us that:
_ A few seconds remain till the end of our byoyomi period.
_ We lost a byoyomi period
_ We lost on time

**Comment by sanderland** [comment]
this needs a nice set of sound files

**Comment by sanderland** [comment]
stone placement sound also requested
